### PR TITLE
Add summit program

### PR DESCRIPTION
**Pull request by bbrcknl**

### CODE REVIEW

File: content/news/2023.pml [hunk 1]
```diff
@@ -54,6 +54,24 @@ Older News: <%=reflist%>
...
+  16 October 2023: The videos and slides of the seL4 summit 2023 are available online
+</div>
+<div>
+  <a href="/Foundation/Summit/2023">
+    <img src="/images/sel4-summit-logo.svg"
+    style="width: 20%;  padding-left:10px;  float:right"
+    alt="seL4 summit" />
+  </a>
+  <p>
+    Videos of the seL4 summit 2023 are now available on the <a href="https://www.youtube.com/@seL4" target="_blank">seL4 YouTube channel</a>! Links and slides can be found on the summit <a href="../Foundation/Summit/2023/program">Program</a> and <a href="../Foundation/Summit/2023/abstracts2023">Abstracts</a> pages. Thanks to all the speakers for making the seL4 summit 2023 a great success!
```
Comment (june-andronick) [inline content/news/2023.pml:69]: Add a link for "seL4 summit 2023"

File: content/Foundation/Summit/2023/home.pml [hunk 2]
```diff
@@ -17,21 +17,22 @@ Minneapolis, USA
 <div class="summit_div">
 <div class="highlight">
 <h4></h4>
-<h4>Latest News: <a href="#summit-sponsors">Sponsors</a></h4>
+<h4>Latest News: <a href="#summit-abstracts">Videos, Slides and Abstracts</a></h4>
```
Comment (june-andronick) [inline content/Foundation/Summit/2023/home.pml:20]: I would already remove this "Latest News" line. This was the page is ready to stay the way it is

File: content/Foundation/Summit/2023/home.pml [hunk 3]
```diff
@@ -17,21 +17,22 @@ Minneapolis, USA
...
+    <!--li><a href="#summit-important-dates">Important Dates</a></li-->
     <li><a href="#summit-about">About</a></li>
     <!--li><a href="../principles/">Principles</a></li-->
-    <li><a href="#summit-registration">Registration</a></li>
+    <!--li><a href="#summit-registration">Registration</a></li-->
     <li><a href="#summit-location">Location and Venue</a></li>
     <li><a href="#summit-keynotes">Keynotes</a></li>
     <li><a href="#summit-panellists">Panellists</a></li>
-    <li><a href="#summit-program">Program</a></li>
+    <li><a href="program">Program</a></li>
```
Comment (june-andronick) [inline content/Foundation/Summit/2023/home.pml:34]: Change to "Program, Abstracts, Slides and Videos" and link to subsection below (and remove "Abstract" in menu).

And do the same in 2022 page

File: content/Foundation/Summit/2023/home.pml [hunk 4]
```diff
@@ -17,21 +17,22 @@ Minneapolis, USA
...
+    <li><a href="abstracts2023">Abstracts</a></li>
```
Comment (june-andronick) [inline content/Foundation/Summit/2023/home.pml:35]: I would remove the menu item "Abstract" here (see above)

File: content/Foundation/Summit/2023/home.pml [hunk 5]
```diff
@@ -94,8 +95,18 @@ Minneapolis, USA
     <li>Networking with other seL4 experts and enthusiasts</li>
     <li>Reception and Dinner</li>
   </ul>
+</div-->
+
+<div class="summit_div">
+  <h3 id="summit-abstracts">
+    Videos, Slides and Abstracts
```
Comment (june-andronick) [inline content/Foundation/Summit/2023/home.pml:102]: I would change the title to "Program, Abstracts, Slides and Videos".
And do the same for the 2022 page.

**Review by june-andronick — APPROVED** [review]
Looks generally good, many thanks @bbrcknl I know it's a lot of work. 

I'd just take the opportunity to clean-up a bit the landing page and make it consistant with the summit 22 one. I've added inline comments in the review. I'm adding here:
- in 2023: remove the keynote abstracts and just have the link to the Abstracts page, like in 2022
- call both 2022 and 2023 "Keynotes Speakers"
- have the menu order the same in 2022 and 2023, and consistent with the sections in the page
- the panellists pictures currently appear one below each other as soon as they don't all 5 fit in a line. It'd be nice to have it like in 2022 where they overflow, eg 5, then 4+1 then 3+2...
- change all future tense to past or present in 2023 (eg "was held" etc)
- I would change the way "All summits" appear as it's hard to spot.  Maybe simply add an empty line below this last bullet point.

**Comment by bbrcknl** [comment]
@june-andronick Thanks for your feedback! Can you please sanity check this before I do the merge?

**Comment by june-andronick** [comment]
> @june-andronick Thanks for your feedback! Can you please sanity check this before I do the merge?

Just a couple of nit-pick:
- I'd keep "Location & Venue" for 2023 and in the text have "The summit was organised by the Linux Foundation" (LF is not really the Host)
- Change 2022's page subtitle to have the date below the location, to look like 2023 :)
- I still have a strange rendering of the 2023 panelists, all pictures one below each other as soon as they don't fit all 5 on 1 line... This works fine for the PC or the sponsors list...
- (if easy: the sponsors would be nicer if centered... but not critical)

**Comment by june-andronick** [comment]
Also @bbrcknl we now have Qiyan's slides, so please add them :)

**Pull request merged**